test/commands/test-cli-helper/test-cli-context.js

```diff
@@ -71,6 +71,7 @@ const preparePlan = ({
   env,
   prompts,
   tokenBehavior,
+  additionnalStep,
```

Comment:
the change in this file is to support the `additionnalStep` keyword, which adds a step at the end of the plan.